neuralop/layers/padding.py

```diff
@@ -84,7 +93,8 @@ def pad(self, x):
 
             out_put_shape = padded.shape[2:]
 
-            out_put_shape = [int(round(i*j)) for (i,j) in zip(self.output_scaling_factor,out_put_shape)]
+            out_put_shape = [round(i * j) for (i, j)
```

Comment:
I know this isn't from this PR by we shouldn't be calling this out_put_shape, just `output_shape` - let's try to remove convoluted names and code from the library